bedrock/firefox/templates/firefox/os/faq.html

```diff
@@ -21,160 +21,206 @@ <h1 class="large">{{_('Firefox OS FAQ')}}</h1>
+                            </a></li>
+                            <li><a href="https://support.mozilla.org/kb/using-camera">
+                                {{_('…take pictures?')}}
+                            </a></li>
+                            <li><a href="https://support.mozilla.org/kb/send-and-manage-email">
+                                {{_('…compose and send an email?')}}
+                            </a></li>
+                        </ul>
+                    </dd>
+                </dl>
+            </div>
+        </section>
+
+        <section>
+            <h2 data-accordion-role="tab">{{_('Features')}}</h2>
+            <div data-accordion-role="tabpanel">
+                <dl class="faq">
+                    <dt><em>{{_('Firefox Marketplace')}}</em></dt>
+                    <dd>{{_('Your go-to source for the best apps for Firefox OS.')}}</dd>
+
+                    <dt><em>{{_('Firefox browser')}}</em></dt>
+                    <dd>{{_('The fast, smart, safe way to get the best of the Web.')}}</dd>
+
+                    <dt><em>{{_('Deep integration with social media')}}</em></dt>
+                    <dd>{{_('Full access to every Twitter and Facebook update from wherever you are.')}}</dd>
+
+                    <dt><em>{{_('Camera')}}</em></dt>
+                    <dd>{{_('With built-in style filters for fun, creative shots.')}}</dd>
+
+                    <dt><em>{{_('FM Radio')}}</em></dt>
+                    <dd>{{_('All the audio you want on the go, all on one device.')}}</dd>
+
+                    <dt><em>{{_('Maps')}}</em></dt>
+                    <dd>{{_('Includes directions, traffic and local transit information.')}}</dd>
+                </dl>
+            </div>
+        </section>
+
+        <section>
+            <h2 data-accordion-role="tab">{{_('Privacy and Security')}}</h2>
+            <div data-accordion-role="tabpanel">
+                <dl class="faq">
+                    <dt>
+                    {% if l10n_has_tag('fxos_faq_updates') %}
+                        {{_('How will Firefox OS keep me safe while I browse?')}}
+                    {% endif %}
+                    </dt>
+                    <dd>{{_('Mozilla was the first to implement the Do Not Track privacy feature.')}}
+                    {% trans url='https://support.mozilla.org/kb/how-turn-do-not-track-feature-firefox-os' %}
+                    Firefox also features customizable security settings to manage passwords, history, private data, cookies, site options and add-ons. <a href="{{url}}">Learn more.</a>
+                    {% endtrans %}
+                    </dd>
+                </dl>
+            </div>
+        </section>
+
+        <section>
+            <h2 data-accordion-role="tab">{{_('Developer Questions')}}</h2>
+            <div data-accordion-role="tabpanel">
+                <p>{% trans url='https://developer.mozilla.org/docs/Mozilla/Firefox_OS' %}
```

Comment:
Please note that every change like this, on a string that is not new (i.e. behind a tag), is going to hurt l10n. So, please don't change them ;-)